cypress/integration/podcasts.spec.js

```diff
@@ -0,0 +1,15 @@
+describe('podcasts', () => {
+  beforeEach(() => {
+    cy.server();
+    cy.visitAndWaitFor('/podcasts');
+    cy.get('h1').should('have.text', 'Podcasts');
+  });
+
+  it('renders many podcast cards', () => {
+    cy.get('[data-testid="podcast-card"]').should('have.length.greaterThan', 30);
+  });
+
+  it('checks if audio is playing', () => {
+    cy.audio('episode.source').should('be.playing', true);
```

Comment:
So this is a good assertion, but I feel like you want to make it after you click on the play button of an episode, no?